cmd/erasure-server-pool-rebalance.go

```diff
@@ -496,23 +549,26 @@ func (z *erasureServerPools) rebalanceBucket(ctx context.Context, bucket string,
 	rcfg, _ := getReplicationConfig(ctx, bucket)
 
 	pool := z.serverPools[poolIdx]
+
 	const envRebalanceWorkers = "_MINIO_REBALANCE_WORKERS"
 	wStr := env.Get(envRebalanceWorkers, strconv.Itoa(len(pool.sets)))
 	workerSize, err := strconv.Atoi(wStr)
 	if err != nil {
 		logger.LogIf(ctx, fmt.Errorf("invalid %s value: %s err: %v, defaulting to %d", envRebalanceWorkers, wStr, err, len(pool.sets)))
 		workerSize = len(pool.sets)
+	} else {
+		// each set get its own thread separate from the concurrent
+		// objects/versions being decommissioned.
+		workerSize += len(pool.sets)
```

Comment:
the comment and the code does not seem to be exact, there += len(pools.sets), which means by default decom uses len(pools.sets)*2, because by default wStr has len(pool.sets)

Reply:
I will change to behave accordingly